### ISSUE TITLE
Support hiredis 0.6.0

### ISSUE THREAD

**Issue by romanbsd** [mloughran/em-hiredis#39]
It was recently released with no API changes as far as I can tell.
https://github.com/redis/hiredis-rb/compare/v0.5.2...v0.6.0

**Comment by jlecour** [comment]
hiredis `0.6` has been around for quite some time, and now that ActionCable (from Rails) is depending on `em-hiredis` we have a dependency conflict.

```
Bundler could not find compatible versions for gem "hiredis":
  In Gemfile:
    hiredis (~> 0.6)

    rails (= 5.0.0.beta1) was resolved to 5.0.0.beta1, which depends on
      actioncable (= 5.0.0.beta1) was resolved to 5.0.0.beta1, which depends on
        em-hiredis (~> 0.3.0) was resolved to 0.3.0, which depends on
          hiredis (~> 0.5.0)
```

**Comment by badboy** [comment]
Now that Rails seems to use it and the new version (0.6.1) works without problems on Windows (pure-Ruby-fallback), it might be nice to upgrade em-hiredis as well.

**Comment by francesco-loreti** [comment]
Yes please upgrade to 0.6.1 ...

**Comment by steakknife** [comment]
Merging #48 and cutting a release could fix this.

**Comment by steakknife** [comment]
Ouch, it's still not actually fixed in released gems... Github might've interpreted "Fixes" as an auto-closing verb.  My bad.

**Comment by mloughran** [comment]
Yes I noticed that. Will get round to releasing in the next day or so. Actually having merged I think it's better to leave the dependency as ~> 0.6.0 so I'll revert before releasing. I don't think there's any reason to prefer 0.6.1, so better to leave the dependency loose. Thanks for the prod!

**Comment by badboy** [comment]
Only 0.6.1 will install flawlessly on Windows

**Comment by sgrif** [comment]
Just a note that this is still disrupting Rails. Can we cut a release soon?

**Comment by mloughran** [comment]
I have just released 0.3.1 which requires `~> 0.6.0`. Hope this helps.

@badboy thanks – however keeping the dependency loose shouldn't cause issues; anyone running a clean bundle install or bundle update will get 0.6.1 assuming they have no other hard dependencies on 0.6.0.

**Comment by steakknife** [comment]
Woohoo.  Least surprise FTW.  Thanks.
